docs/getting-started/13-trigger-function-with-event.md

```diff
@@ -3,83 +3,86 @@ title: Trigger the Function with an event
-- **spec.filter.attributes.type** points to the event type to which you want to subscribe the Function. In this example, it is `order.deliverysent`.
+The event type is composed of the following components:
+- Prefix: `sap.kyma.custom`
+- Application: `commerce`
+- Event: `order.created`
+- Version: `v1`
 
-2. Check that the Trigger CR was created and is ready. This is indicated by its status equal to `True`:
+2. Check that the Subscription CR was created and is ready. This is indicated by its status equal to `true`:
 
-  ```bash
-  kubectl get trigger orders-function -n orders-service -o=jsonpath="{.status.conditions[2].status}"
-  ```
+   ```bash
+   kubectl get subscriptions.eventing.kyma-project.io orders-sub -n orders-service -o=jsonpath="{.status.ready}"
+   ```
 
-    </details>
-    <details>
-    <summary label="console-ui">
-    Console UI
-    </summary>
+   </details>
+<details>
+<summary label="console-ui">
+Console UI
+</summary>
 
 1. From the drop-down list in the top navigation panel, select the `orders-service` Namespace.
 
 2. Go to **Workloads** > **Functions** in the left navigation panel and navigate to `orders-function`.
 
-3. Once in the Function's details view, switch to the **Configuration** tab and select **Add Event Trigger** in the **Event Triggers** section.
+3. Once in the Function's details view, switch to the **Configuration** tab and select **Create Event Subscriptions** in the **Event Subscriptions** section.
 
-4. Once the pop-up box opens, find the `order.deliverysent` event with the `v1` version from the `commerce-mock` application. Check it on the list and select **Add**.
+4. Once the pop-up box opens, find the `order.created.v1` event with the `v1` version from the `commerce-mock` application. Check it on the list and select **Add**.
 
-A message confirming that the event trigger was created will appear in the **Event Triggers** section in the Function's details view.
+A message confirming that the Subscription was created will appear in the **Event Subscriptions** section in the Function's details view.
 
-    </details>
+  </details>
 </div>
 
-### Test the Trigger
+### Test the Event delivery
 
 To send events from Commerce mock to `orders-function`, follow these steps:
 
 1. Access Commerce mock at `https://commerce-orders-service.{CLUSTER_DOMAIN}.` or use the link under **Host** in the **Discovery and Network** > **API Rules** view in the `order-service` Namespace.
 
 2. Switch to the **Remote APIs** tab, find **SAP Commerce Cloud - Events**, and select it.
 
-3. From the **Event Topics** drop-down list, select the `order.deliverysent.v1` event type. In the details of the printed event, change **orderCode** to `987654321` and select **Send Event**.
+3. From the **Event Topics** drop-down list, select the `order.created.v1` event type. In the details of the printed event, change **orderCode** to `987654321` and select **Send Event**.
```

Comment:
Other parts of the GS guides mention "deliverysent"

Reply:
Thank you, I changed it in 93202d8